src/core/config/config_module.rs

```diff
@@ -96,6 +96,18 @@ impl Deref for ConfigModule {
     }
 }
 
+impl ConfigModule {
+    /// Renders current config to graphQL string
+    pub fn to_sdl(self) -> String {
+        crate::core::document::print(self.into())
+    }
+
+    /// Normalizes current config with default settings
+    pub fn normalize_default(self) -> Valid<Self, String> {
+        self.transform(UnionInputType.pipe(AmbiguousType::default()))
```

Comment:
Can also add `remove_unused` over here.

Reply:
let's do it separately because some of the tests get broken when adding it